tests/typography/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js">
+
+<head>
+  <meta charset="utf-8" />
+  <title>The University of British Columbia | TEXT TEST</title>
+  <meta name="description" content="TEST" />
+  <meta name="author" content="Someone" />
+  <meta name="handheldfriendly" content="true" />
+  <meta name="mobileoptimized" content="320" />
+  <meta name="viewport" content="width=device-width" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <link rel="shortcut icon" href="https://cdn.ubc.ca/clf/7.0.4/img/favicon.ico" />
+  <link rel="stylesheet" href="../../dist/css/styles-minimal.min.css" />
+  <style>
+    /* https://basehold.it - baseline bg image - the 24 denotes the line height - change if needed */
+    
+    .grid-wrapper {
+      background-image: url('http://basehold.it/i/24');
```

Comment:
I think we should use https assets in demos when they're available.

This redirects to the https version anyway, but let's be explicit.

Reply:
Fixed